Docs/user_guide/coordinate_systems.md

```diff
@@ -0,0 +1,268 @@
+right, the $j$ axis to the bottom and the $k$ axis backwards.
+
+In addition to the intensity value of each voxel $(i j k)$ the origin and
+spacing of the anatomical coordinates are stored too.
+
+- The origin represents the position of the first voxel (0, 0, 0) in the
+  anatomical coordinate system, e.g. (100, 50, -25) mm.
+- The spacing specifies the distance between voxels along each axis, e.g.
+  (1.5, 0.5, 0.5) mm.
+
+The following 2D example shows the meaning of origin and spacing:
+
+![image_coordinates](https://github.com/Slicer/Slicer/releases/download/docs-resources/image_coordinates.png){.bg-warning w=300px align=center}
+
+Using the origin and spacing, the corresponding position of each (image
+coordinate) voxel in anatomical coordinates can be calculated.
+
+## Image transformation
+
+The transformation from an image space vector $(i j k)'$ to an anatomical
+space vector $\vec{x}$ is an affine transformation, consists of a linear
+transformation $\mathbf{A}$ followed by a translation $\vec{t}$.
+
+$$
+\vec{x} = A \begin{pmatrix} i & j & k \end{pmatrix}' + \vec{t}
+$$
+
+The transformation matrix $\mathbf{A}$ is a $3 \times 3$ matrix and carries
+all information about space directions and axis scaling.
+
+$\vec{t}$ is a $3 \times 1$ vector and contains information about the
+geometric position of the first voxel.
+
+$$
+\begin{pmatrix} x_1 \\ x_2 \\ x_3 \end{pmatrix} =
+\begin{pmatrix} A_{11} & A_{12} & A_{13} \\ A_{21} & A_{22} & A_{23} \\ A_{31} & A_{32} & A_{33} \end{pmatrix}
+\begin{pmatrix} i \\ j \\ k \end{pmatrix} + \begin{pmatrix} t_1 \\ t_2 \\ t_3 \end{pmatrix}
+$$
+
+The last equation shows that the linear transformation is performed by a
+matrix multiplication and the translation by a vector addition. To represent
+both, the transformation and the translation, by a matrix multiplication an
+augmented matrix must be used. This technique requires that the matrix
+$\mathbf{A}$ is augmented with an extra row of zeros at the bottom, an extra
+column-the translation vector-to the right, and a $1$ in the lower right
+corner. Additionally, all vectors have to be written as homogeneous
+coordinates, which means that a $1$ is augmented at the end.
+
+$$
+\begin{pmatrix} x_1 \\ x_2 \\ x_3 \\ 1 \end{pmatrix} =
+\begin{pmatrix} A_{11} & A_{12} & A_{13} & t_1 \\ A_{21} & A_{22} & A_{23} & t_2 \\
+                A_{31} & A_{32} & A_{33} & t_3 \\ 0 & 0 & 0 & 1 \end{pmatrix}
+\begin{pmatrix} i \\ j \\ k \\ 1 \end{pmatrix}
+$$
+
+Depending on the used anatomical space (LPS or RAS) the $4 \times 4$ matrix is
+called **IJKtoLPS**- or **IJKtoRAS**-matrix, because it represents the
+transformation from IJK to LPS or RAS.
+
+## 2D example or calculating an *IJtoLS*-matrix
+
+The following figure shows the anatomical space with an L(P)S basis on the
+left and the corresponding image coordinates on the right.
+
+![IJtoLS](https://github.com/Slicer/Slicer/releases/download/docs-resources/IJtoLS.png){.bg-warning w=550px align=center}
```

Comment:
```suggestion
![IJtoLS](https://github.com/Slicer/Slicer/releases/download/docs-resources/coordinate_systems_IJtoLS.png){w=550px align=center}
```

Reply:
Done in [0b7a79e](https://github.com/Slicer/Slicer/commit/0b7a79ed1b791335814ae637ee267ba8edf3750b).